packages/dev/docs/pages/releases/2022-10-04.mdx

```diff
@@ -0,0 +1,282 @@
+{/* Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {BlogPostLayout, Hero} from '@react-spectrum/docs';
+export default BlogPostLayout;
+
+---
+description: We have been busy this month with new components and plenty of drag and drop updates. Drag and drop has moved into RC status in this release, and includes many API improvements to make common experiences even easier to build. See below for more information on drag and drop below for more information. We have also added Badge and LabeledValue in this release. Badges are handy for showing a small amount of color-categorized metadata and LabeledValue is for displaying a non-editable values with a label such as in a properties panel. It also formats numbers, dates, times, and lists according to the user's locale. CheckBoxGroup and RadioGroup have also been updated to include HelpText and ContextualHelp support has been added to all form components.
+date: 2022-10-04
+---
+
+# October 4, 2022 Release
+We have been busy this month with new components and plenty of drag and drop updates. Drag and drop has moved into RC status in this release, and includes many API improvements to make common experiences even easier to build. See [below](#drag-and-drop-updates) for more information on drag and drop below for more information. We have also added [Badge](/react-spectrum/Badge.html) and [LabeledValue](/react-spectrum/LabeledValue.html) in this release. Badges are handy for showing a small amount of color-categorized metadata and LabeledValue is for displaying a non-editable values with a label such as in a properties panel. It also formats numbers, dates, times, and lists according to the user's locale. [CheckBoxGroup](/react-spectrum/CheckboxGroup.html#help-text) and [RadioGroup](react-spectrum/RadioGroup.html#help-text) have also been updated to include HelpText and [ContextualHelp](/react-spectrum/TextField.html#contextual-help) support has been added to all form components.
```

Comment:
```suggestion
We have been busy this month with new components and plenty of drag and drop updates. Drag and drop has moved into RC status in this release, and includes many API improvements to make common experiences even easier to build. See [below](#drag-and-drop-updates) for more information on drag and drop below for more information. We have also added [Badge](/react-spectrum/Badge.html) and [LabeledValue](/react-spectrum/LabeledValue.html) in this release. Badges are handy for showing a small amount of color-categorized metadata and LabeledValue is for displaying non-editable values with a label such as in a properties panel. It also formats numbers, dates, times, and lists according to the user's locale. [CheckBoxGroup](/react-spectrum/CheckboxGroup.html#help-text) and [RadioGroup](react-spectrum/RadioGroup.html#help-text) have also been updated to include HelpText and [ContextualHelp](/react-spectrum/TextField.html#contextual-help) support has been added to all form components.
```

Reply:
Also, I'm not sure if these links will work in prod, so it might be better to update them to the full link (i.e. `https://react-spectrum.adobe.com/react-spectrum/Badge.html`)

Reply:
They should be ok but I will update just to be safe